js/lesson-05.js

```diff
@@ -0,0 +1,125 @@
+// !!!  ДЗ 10. Масив
+// Створити масив, довжину та елементи якого задає користувач.
+// Відсортувати масив за зростанням.
+// Видалити елементи з масиву з 2 по 4 (включно!).
+// У міру змін виводити вміст масиву на сторінку.
+
+// !!! Рішення
+
+// const arrayLength = Number(prompt('Введіть довжину для масиву')); 
+// if (arrayLength === 0) {
+//    alert (`Ви не ввели довжину для масиву`)
+// } else if(isNaN(arrayLength)) {
+//    alert (`Ви ввели не числове значення для довжини масиву`)
+// } else {
+//    const userArray = new Array(arrayLength);
+//    console.log(userArray);
+   
+//    for (let i=0; i<arrayLength; i++) {
+//       userArray[i] = prompt('Введіть дані для масиву','');
+//    };
+//    console.log(`Ви ввели такі данні для масиву: ${userArray}`)
+   
+//    const sortedArray = userArray.sort();
```

Comment:
В сортування бажано додати параметри, інакше воно буде сортувати не за числом, а поелементно. Хоча це не є помилкою.

```suggestion
//    const sortedArray = userArray.sort().sort((a, b) => a - b);
```